rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -186,14 +321,60 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    MetadataInformationResponse:
+      type: object
+      required:
+        - transformationStatus
+        - metadataInformations
+      properties:
+        transformationStatus:
+          $ref: '#/components/schemas/Status'
+        errorMessage:
+          description: An optional error message in the case the transformation failed.
+          $ref: '#/components/schemas/ErrorMessage'
+        metadataInformations:
+          type: array
+          items:
+            $ref: '#/components/schemas/MetadataInformation'
+    MetadataInformation:
+      type: object
+      required:
+        - metadataType
+        - transformationStatus
+      properties:
+        metadataType:
+          $ref: '#/components/schemas/MetadataType'
+        transformationStatus:
+          $ref: '#/components/schemas/Status'
+        errorMessage:
+          description: An optional error message in the case the transformation failed.
+          $ref: '#/components/schemas/ErrorMessage'
+        location:
+          description: The URI for where the metadata is located. NOTE this will not be populated until the transformationStatus is Complete.
+          type: string
+          format: uri
+          example: 'https://www.example.com/mis/30f14c6c1fc85cba12bfd093aa8f90e3'
+        transformedTimestamp:
+          type: string
+          format: date-time
+          description: The server time when the metadata was generated. NOTE this will not be populated until the transformationStatus is Complete.
+          example: 2019-05-09T14:47:13.101+0000
+    MetadataType:
+      description: The type of metadata.
+      type: string
+      enum: [irm]
+    Status:
+      description: The status of the operation.
+      type: string
+      enum: [In Progress, Done, Failed]
     TransformRequest:
       type: object
       required:
         - location
         - mimeType
         - metacardLocation
       properties:
-        location:
+        currentLocation:
           description: >
             This URI must support HTTP GET to retrieve the file and HTTP PUT with attachment at
```

Comment:
❗️ This is no longer true. The URI provided should only allow us to retrieve the product.